demo/index.html

```diff
@@ -146,25 +146,30 @@
 
           videos: {
             type: Array,
-            value: []
+            value: function() {
+              return [];
+            }
           },
 
           data: {
             type: Object,
-            value: {
-              page: '/search/'
+            value: function() {
+              return {
+                page: '/search/'
+              };
             }
-          }
+          },
```

Comment:
Trailing commas are great! Makes it easier to rearrange or comment out entries in an array or object literal.